RibonDAO/app
# feat: add firebase analytics and developer build configs

# Description

- Added configs for building the app as dev-client
- Added configs for firebase analytics

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- yarn install
- eas build --profile development-simulator --platform ios (or get the app with me)
- open the app on ios simulator
- expo start --dev-client
